plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -308,7 +308,7 @@
         <mavenCoordinates
               artifactId="google-cloud-datastore"
               groupId="com.google.cloud"
-              version="1.0.1" />
+              version="LATEST" />
```

Comment:
Looking at [google-cloud-datastore](https://repo1.maven.org/maven2/com/google/cloud/google-cloud-datastore/1.4.0/google-cloud-datastore-1.4.0.pom), it also requires:
  - `com.google.cloud.datastore:datastore-v1-protos:1.3.0`
  - `com.google.cloud.datastore:datastore-v1-proto-client:1.3.0`
  - `com.google.api.grpc:grpc-google-common-protos:0.1.15`